jobs/awslogs/templates/bin/pre-start

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+for I in `find  /var/vcap/sys/log/ -mindepth 1 -type d`
```

Comment:
I'm going to be super pedantic and suggest you removing the trailing slash in the path, so we get file names like `/var/vcap/sys/log/foo/bar.log` and not `/var/vcap/sys/log//foo/bar.log`.